### ISSUE TITLE
create-directus-extension is currently broken

### ISSUE THREAD

**Issue by Huhspaniel** [directus/directus#17100]
### Describe the Bug

On a clean install of node and npm, this is the output after running `npm init directus-extension`:

```
This utility will walk you through creating a Directus extension.

/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/inquirer/lib/prompts/base.js:81
    throw new Error('You must provide a `' + name + '` parameter');
          ^

Error: You must provide a `choices` parameter
    at ListPrompt.throwParamError (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/inquirer/lib/prompts/base.js:81:11)
    at new ListPrompt (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/inquirer/lib/prompts/list.js:26:12)
    at PromptUI.fetchAnswer (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/inquirer/lib/ui/prompt.js:105:25)
    at doInnerSub (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/mergeInternals.js:22:31)
    at outerNext (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/mergeInternals.js:17:70)
    at OperatorSubscriber._this._next (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/OperatorSubscriber.js:33:21)
    at OperatorSubscriber.Subscriber.next (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/Subscriber.js:51:18)
    at /home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/mergeInternals.js:28:28
    at OperatorSubscriber._this._next (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/OperatorSubscriber.js:33:21)
    at OperatorSubscriber.Subscriber.next (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/Subscriber.js:51:18)
npm ERR! code 1
npm ERR! path /home/daniel/code
npm ERR! command failed
npm ERR! command sh -c cde "webflow-api"

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/daniel/.npm/_logs/2023-01-11T20_39_40_247Z-debug.log
```

I am on Linux.

### To Reproduce

1) Install nvm: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash`
2) Install version 16.13.0 of node: `nvm install 16.13.0`
3) Execute the create-directus-extension utility: `npm init directus-extension`

### Errors Shown

```
/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/inquirer/lib/prompts/base.js:81
    throw new Error('You must provide a `' + name + '` parameter');
          ^

Error: You must provide a `choices` parameter
    at ListPrompt.throwParamError (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/inquirer/lib/prompts/base.js:81:11)
    at new ListPrompt (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/inquirer/lib/prompts/list.js:26:12)
    at PromptUI.fetchAnswer (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/inquirer/lib/ui/prompt.js:105:25)
    at doInnerSub (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/mergeInternals.js:22:31)
    at outerNext (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/mergeInternals.js:17:70)
    at OperatorSubscriber._this._next (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/OperatorSubscriber.js:33:21)
    at OperatorSubscriber.Subscriber.next (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/Subscriber.js:51:18)
    at /home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/mergeInternals.js:28:28
    at OperatorSubscriber._this._next (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/operators/OperatorSubscriber.js:33:21)
    at OperatorSubscriber.Subscriber.next (/home/daniel/.npm/_npx/9ea216cce9aeac9a/node_modules/rxjs/dist/cjs/internal/Subscriber.js:51:18)
npm ERR! code 1
npm ERR! path /home/daniel/code
npm ERR! command failed
npm ERR! command sh -c cde "webflow-api"

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/daniel/.npm/_logs/2023-01-11T20_39_40_247Z-debug.log
```

### What version of Directus are you using?

N/A

### What version of Node.js are you using?

16.13.0

### What database are you using?

N/A

### What browser are you using?

N/A

### How are you deploying Directus?

N/A

**Comment by rijkvanzanten** [comment]
Fixed in https://github.com/directus/directus/pull/17098

**Comment by Huhspaniel** [comment]
> Fixed in #17098

I am still able to reproduce this issue.

**Comment by rijkvanzanten** [comment]
> > Fixed in #17098
> 
> I am still able to reproduce this issue.

Merged ≠ Released, I'll push out a patch in a jiffy 👍🏻
